ocaml/parsing/ast_iterator.ml

```diff
@@ -177,7 +171,9 @@ module T = struct
     | Ptyp_var _ -> ()
     | Ptyp_arrow (_lab, t1, t2) ->
         sub.typ sub t1; sub.typ sub t2
-    | Ptyp_tuple tyl -> List.iter (sub.typ sub) tyl
+    | Ptyp_tuple tyl ->
+        (* CR labeled tuples: Eventually iterators may want to see the labels *)
```

Comment:
True, but there are a bunch of ignored labels elsewhere here. I don't think this comment is necessary.